### PR TITLE
Second Request

### PR DESCRIPTION
**Pull request by nextcode007**

### CODE REVIEW

File: render.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+from xml.etree import ElementTree as ET
+
+ns = {'ns': 'http://www.transunion.com/namespace/pfs/v4'}
+
+def render_print_image(data):
+    root = ET.fromstring(data)
+        
+    result = "TRANSUNION CREDIT REPORT\n\n"
```
Comment (daronstinnett) [inline render.py:8]: how does this produce a line that is 77 characters?

File: render.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    result += "<FOR>           <SUB NAME>          <MKT SUB>  <INFILE>    <DATE>     <TIME> \n"
```
Comment (daronstinnett) [inline render.py:9]: do not put linebreaks in list items. `all(len(l) == 77 for l in result)` should always be True

File: render.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            result += "***Warnings***"  + "\n"
+            flag = True
+        if len(str(warning.text)) > 77:
+            result += "***" + str(warning.text)[0:74] + "\n"
+            result += str(warning.text)[74:] + "***" + "\n"
+        else:
+            result += "***" + str(warning.text) + "***" + "\n"
+    result += "-" * 77 + "\n"
+    result += "M O D E L  P R O F I L E          * * * A L E R T * * *" + "\n"
+    result += "***RECOVERY MODEL 1.0 SCORE +675 : ***" + "\n"
```
Comment (daronstinnett) [inline render.py:58]: The static text in this section would mislead a person looking at this report. It should be removed if the data is not available in the xml.

File: render.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    result += "***NEW ACCT MODEL 2.0 ALERT: SCORE +731 : 010, 017, 031, 058 *** IN" + "\n"
+    result += "***ADDITION TO THE FACTORS LISTED ABOVE, THE NUMBER OF INQUIRIES ON THE" + "\n"
+    result += "***CONSUMER'S CREDIT FILE HAS ADVERSELY AFFECTED THE CREDIT SCORE." + "\n"
+    result += "-" * 77 + "\n"
+    result += "T R A D E S" + "\n"
```
Comment (daronstinnett) [inline render.py:63]: The public records and collections sections are missing
image

**Review by daronstinnett — CHANGES_REQUESTED** [review]

**Comment by nextcode007** [comment]
When to show or hide Model Profile?
What is the tag for that?

**Pull request closed without merging**